index-es.html

```diff
@@ -82,9 +82,9 @@ <h2>La Versión Menos Corta</h2>
 
 <p>Si estás sufriendo algún abuso, adviertes abusos sobre otra persona, o tienes alguna otra preocupación, por favor contacta con algún miembro del personal de la conferencia inmediatamente. El personal de la conferencia puede ser fácilmente identificado, ya que vestirán camisetas del evento.</p>
 
-<p>El personal de la conferencia estará contento de ayudar a los participates a contactar con la seguridad del hotel/sala o con las autoridades locales, proveer escolta o asistir de otra manera a aquellos que estén sufriendo abusos para que se sientan seguros mientras dure la conferencia. Valoramos tu asistencia.</p>
+<p>El personal de la conferencia estará atento para ayudar a los participates a contactar con la seguridad del hotel/sala o con las autoridades locales, proveer escolta o asistir de otra manera a aquellos que estén sufriendo abusos para que se sientan seguros mientras dure la conferencia. Valoramos tu asistencia.</p>
```

Comment:
¿Maybe `estará encantado de ayudar`? Sounds more natural

Reply:
Thanks @PBartrina for your comment! Beyond this suggestion improvement, would you approve this translation? 🙂

Reply:
In particular, does it seem up to date with the current main English version?